server/services/config/conditions/v2/conditions.ts

```diff
@@ -47,13 +65,21 @@ export const conditions: ConditionMetadata[] = [
   {
     id: 'ATTEND_ALL',
     text: 'Attend all appointments arranged for you with [INSERT NAME], a [PSYCHIATRIST / PSYCHOLOGIST / MEDICAL PRACTITIONER] and co-operate fully with any care or treatment they recommend.',
-    user_input: 'appointmentName', // need to change?!
+    user_input: 'attendAll',
```

Comment:
had to create a new form as forms aren't versioned and appointmentName is still being used by v1